index.html

```diff
@@ -0,0 +1,191 @@
+  </figure>
+  <figure class="item6">
+    <img src="Resources/soup.jpg" class="imageps hvr-grow" alt="Image 6">
+  </figure>
+
+  </div>
+</div>
+
+<!-- Recetas -->
+
+<div class="recetas" id="recetas">
+  <h2>Nuestras Recetas</h2>
+  
+  <div class="receta-container">
+    
+    <figure class="receta1">
+      <img src="Resources/receta1.jpg" class="imageps">
+    </figure>
+    <figure class="receta1">
+      <img src="Resources/receta2.jpg" class="imageps">
+    </figure>
+    <figure class="receta1">
+      <img src="Resources/receta3.jpg" class="imageps">
+    </figure>    
+  </div>
+
+  <h4>¡mirá nuestras recetas!</h4>
+  <h1>___</h1>
+</div>
+
+<div class="videos">
+  <h2>Videos de Recetas</h2>
+
+  <div class="video-container">
+    <figure class="video">      
+      <iframe class="youtube" width="480" height="315" src="https://www.youtube.com/embed/FTPBYWq_SZ8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>   
+      <h3>Tacos al pastor</h3>
+      <h4>Ingredientes</h4>
+      <ul>
+        <li>Carne de Pollo</li>
+        <li>1 chile de ancho</li>
+        <li>1 chile guajilli</li>
+        <li>1 cebolla</li>
+        <li>1 diente de ajo</li>    
+      </ul>
+   </figure>
+
+    <figure class="video">
+      <iframe class="youtube" width="480" height="315" src="https://www.youtube.com/embed/eG3vS9dDeoc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      <h3>Pechuga de pollo rellena</h3>
+      <h4>Ingredientes</h4> 
+      <ul>
+        <li>4 pizcas pollo en filete</li>
+        <li>1 pizca cebolla en julianas</li>
+        <li>1 pizca pimiento rojo en julianas</li>
+        <li>1 taza champiñón en rebanadas</li>
+        <li>1 diente ajo finamente picado</li>
+        <li>1 pizca sal</li>
+      </ul>
+
+    </figure>
+
+    <figure class="video">
+      <iframe class="youtube" width="480" height="315" src="https://www.youtube.com/embed/xVyMN7do-2E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      <h3>Panqueque de Banana y avena</h3>
+      <h4>Ingredientes</h4>
+      <ul>
+        <li>1 banana madura</li>
+        <li>2 huevos</li>
+        <li>40 gramos de copo de avena</li>
+        <li>Aceite</li>
+      </ul>
+      
+    </figure>
+  </div>
+
+</div>
+
+<div class="instagram-container" id="insta">
+
+  <div class="instagram">
+
+  <div class="instatexto">
+    <h2>Seguinos por Instagram</h2>
+    <p><strong>Follow</strong> y sumate a nuestros seguidores para no  perderte<br>nuestras recetas semanales, y participar de increíbles premios!</p>
+    <h4><strong>Click</strong> en el logo para seguirnos!</h4>
+    <a href="https://www.instagram.com/holapanzasana/">
+      <img class="logoinsta hvr-pulse-grow" src="Resources/instagram.svg"></a>
+      <div class="underline">   </div>
```

Comment:
do u use this div at all? if not plz remove it.. Otherwise I suggest removing the space between the open & close tag or just close it like this `<div class="underline" />`

Reply:
yes! It is a way that I found on internet for putting this line
![image](https://user-images.githubusercontent.com/67518785/89805041-698a0e80-db0b-11ea-9e68-de6ad3f69e95.png)